test/katex-spec.js

```diff
@@ -19,7 +25,19 @@ var getBuilt = function(expr, settings) {
     expect(expr).toBuild(usedSettings);
 
     var parsedTree = parseTree(expr, usedSettings);
-    var built = buildHTML(parsedTree, usedSettings);
+
+    var startStyle = Style.TEXT;
+    if (usedSettings.displayMode) {
+        startStyle = Style.DISPLAY;
+    }
+
+    // Setup the default options
+    var usedOptions = new Options({
+        style: startStyle,
+        size: "size5"
+    });
```

Comment:
:( Having this duplicated all over here makes me sad. Is there a good place this could be consolidated?